configuration/configuration.go

```diff
@@ -45,6 +45,9 @@ func FromFile(filePath string) (Configuration, error) {
 	conf := &Configuration{}
 	err := conf.FromFile(filePath)
 	setValueFromEnv(&conf.Marathon.Endpoint, "MARATHON_ENDPOINT")
+	setValueFromEnv(&conf.Marathon.Zookeeper.Host, "MARATHON_ZK_HOST")
```

Comment:
It is backwards compatible. To use the feature you need to set MARATHON_ZK_HOST, MARATHON_ZK_PATH, and MARATHON_USE_ZK. Otherwise old behavior is used. MARATHON_ENDPOINT is ignored when MARATHON_USE_ZK is set.